.github/workflows/e2e_tests.yml

```diff
@@ -126,7 +115,13 @@ jobs:
 
     strategy:
       matrix:
-        version: ["1.17.14-gke.2600"]
+        include:
+          - version: "1.17.14-gke.2600"
+            channel: "regular"
+          - version: "1.16.15-gke.6000"
+            channel: "stable"
+          - version: "1.18.12-gke.1201"
```

Comment:
nit - can we re-arrange from higher version to lower for easy readability

Reply:
yes, will also add the versions for EKS (and AKS when test pass)